noxfile.py

```diff
@@ -69,7 +69,10 @@ def unit(session):
         "--cov-config=.coveragerc",
         "--cov-report=term",
         os.path.join("tests", "unit"),
-        env={"PSO_DV_CONFIG_HOME": ""},
+        env={
+            "PSO_DV_CONFIG_HOME": "",
+            "PROJECT_ID": os.environ.get("PROJECT_ID", "pso-kokoro-resources"),
```

Comment:
I think this may be causing the nox tests to fail. Is this required for the new tests? Usually within the unit test we get the PROJECT_ID env var if needd

Reply:
I wasn't sure how would be the best way to provide the env variable so I followed something in that sense that we've for other test. But it makes sense, I'll try provide it just for that specific unit test.